### PR TITLE
feat(styles): updated active state for more menu and resolve error

### PR DESCRIPTION
**Pull request by veralygit**
#### What this PR does / why we need it:
- [x] Add "More" on header (Consortium and On Chain Governance)
- [x] Add Caret Down/Up on active state
- [x] Add active state when one of the items in "more" is selected
- [x] Fix warnings/errors

#### Which issue(s) does this PR fixes?:

Fixes #

#### Sample Links & Screenshots:

Link: 
<details>
<summary>Desktop Screenshot</summary>

</details>
<details>
<summary>Mobile Screenshot</summary>

</details>

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on multiple web browsers
- [ ] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [ ] No console errors
- [ ] Unit tests*
- [ ] Added e2e tests*

### CODE REVIEW

File: src/layouts/components/Header.tsx [hunk 1]
```diff
@@ -327,43 +327,72 @@ function MenuItems(): JSX.Element {
...
+                "dark:text-dark-50 text-primary-500": isItemClicked,
+              }
+            )}
+          >
+            More
+            {open ? (
+              <IoChevronDown className="ml-2" size={20} />
+            ) : (
+              <IoChevronUp className="ml-2" size={20} />
+            )}
```
Comment (pierregee) [inline src/layouts/components/Header.tsx:350]:
```suggestion
            {open ? (
               <IoChevronUp className="ml-2" size={20} />
            ) : (
             <IoChevronDown className="ml-2" size={20} />
            )}
```

File: src/layouts/components/Header.tsx [hunk 2]
```diff
@@ -327,43 +327,72 @@ function MenuItems(): JSX.Element {
...
+                },
+              ].map((item, index) => {
+                if (router.pathname === item.link) {
+                  setIsItemClicked(true);
+                } else {
+                  setIsItemClicked(false);
+                }
+                return (
+                  <Menu.Item key={index}>
+                    <>
```
Comment (pierregee) [inline src/layouts/components/Header.tsx:374]: can remove fragment `<>`?

File: src/layouts/components/Header.tsx [hunk 3]
```diff
@@ -327,43 +327,72 @@ function MenuItems(): JSX.Element {
...
+            leaveTo="transform scale-95 opacity-0"
+          >
+            <Menu.Items className="absolute m-4 min-w-max flex flex-col divide-y bg-white border rounded-lg border-gray-200 dark:border-gray-700 dark:bg-gray-700">
+              {[
+                {
+                  name: "Consortium",
+                  link: "/consortium/asset_breakdown",
+                },
+              ].map((item, index) => {
+                if (router.pathname === item.link) {
```
Comment (pierregee) [inline src/layouts/components/Header.tsx:367]: need to update logic, proof of assets is not covered for active state 🤔 
image

**Review by pierregee — CHANGES_REQUESTED** [review]

File: src/layouts/components/Header.tsx (hunk 3 above)
Comment (veralygit) [reply src/layouts/components/Header.tsx]: updated logic for this however still in the midst of fixing warning 
`Warning: Cannot update a component (`MoreDropdown`) while rendering a different component (`ForwardRef`).`

Comment (veralygit) [reply src/layouts/components/Header.tsx]: fixed warning errors and moved dropdown links to a separate variable

**Review by pierregee — APPROVED** [review]

**Pull request merged**